packages/marko-web-theme-monorail/components/blocks/section-feed.marko

```diff
@@ -6,10 +6,8 @@ import setSectionNameFromLabel from "../../utils/set-section-name-from-label";
 $ const queryName = input.queryName ? input.queryName : "website-scheduled-content";
 $ const queryParams = {
   ...input.queryParams,
-  ...(queryName === "website-scheduled-content") && {
-    sectionAlias: input.alias,
-    queryFragment,
-  }
+  sectionAlias: input.alias,
```

Comment:
@Shinsina To be safe here we should not remove this logic, but rather add a second one for website-optioned-content. SO,

```markojs
...(queryName === "website-scheduled-content") && {
  sectionAlias: input.alias,
  queryFragment,
},
...(queryName === "website-optioned-content") && {
  sectionAlias: input.alias,
  optionName: input.optionName,
  queryFragment,
}
```